assembly/package-res/README_OSX.txt

```diff
@@ -1,14 +1,6 @@
 
 Installation instructions:
 
-You can launch Spoon by navigating to the folder this "read me" file is located in and double clicking on "Data Integration 32-bit" 
-or "Data Integration 64-bit" application icon, depending on your system.
+You can launch Spoon by navigating to the folder this "read me" file is located in and double clicking on the "Data Integration" application icon, depending on your system.
 
-Problems can be reported to our JIRA system:
-
-  http://jira.pentaho.org
-
-
-All the best
-
-The development team
+If you need any assistance, please use http://forums.pentaho.com, or contact Pentaho Support for assistance.
```

Comment:
Not sure about removing information about JIRA. I suggest changing your line to this:

Please use http://forums.pentaho.com for questions. If you find a defect, please report it in our JIRA system at http://jira.pentaho.com. If you are a subscription customer, please contact Pentaho Support for assistance.

Or something like that.